README.md

```diff
@@ -238,6 +238,10 @@ FlightOfferSearch[] flightOffersSearches = amadeus.shopping.flightOffersSearch.g
 // body can be a String version of your JSON or a JsonObject
 FlightOfferSearch[] flightOffersSearches = amadeus.shopping.flightOffersSearch.post(body);
 
+// Flight Order Management
+// The flightOrderID comes from the Flight Create Orders (in test environment it's temporary)
+com.amadeus.resources.FlightOrder order = amadeus.booking.flightOrder("eJzTd9f3NjIJdzUGAAp%2fAiY=").get();
```

Comment:
Can we avoid using fully-qualified name of the class, like in the rest of the examples?
The argument to amadeus.booking.flightOrder() is a flightOrderID - this fact is not very obvious.